spec/rails_helper.rb

```diff
@@ -108,17 +107,6 @@
     create(:setting, mission: nil)
   end
 
-  # Print browser logs to console if they are non-empty.
```

Comment:
same as above, this isn't supported in the new version of the webdrivers.

is this required or useful in other contexts or is it okay to delete?

Reply:
this is fairly important for debugging sometimes -- it's the only place where JS logs (like console.log) get printed when running specs. do you know what specifically doesn't work in the newer version?

Reply:
hmm, this was quite a while ago so I'm not exactly sure what was going wrong or the text of the error messages I was getting.

Reply:
Created new issue in redmine and reverted commit!